AlertaCovid/src/components/atoms/ResultAtoms/BtnResult.js

```diff
@@ -1,15 +1,28 @@
 import React from 'react';
-import {Text, TouchableOpacity} from 'react-native';
+import {Text, TouchableOpacity, Alert} from 'react-native';
 import {ResultStyles} from '../../../styles';
 import {constants} from '../../../utils/constants';
 import {test_positive} from '../../../redux/actions/HealthActions';
 import {connect} from 'react-redux';
 import {checkTest} from '../../../utils/functions';
-const Submit = ({testResult, test_positive}) => {
+const Submit = ({testResult, test_positive, navigation}) => {
+  const Alert = () => {
+    Alert.alert(
+      constants.successfully,
+      constants.message,
+      [
+        {
+          text: 'ok',
+        },
+      ],
+      {cancelable: true},
+    );
+  };
+
   return (
     <TouchableOpacity
       style={ResultStyles.btnContainerSubmit}
-      onPress={() => checkTest(testResult, test_positive)}>
+      onPress={() => Alert(checkTest(testResult, test_positive))}>
```

Comment:
You are using a reserved word to name the funtion, this is not the best practice to name a variable I recommend you change the name.